docs/supported_platforms.md

```diff
@@ -5,7 +5,7 @@ Shadow officially supports the following platforms:
   + Ubuntu 18.04 and 20.04
   + Debian 10
   + Fedora 33
-  + CentOS 7 and 8
+  + CentOS 8
```

Comment:
I think we should add some specific details here to explain that not only is CentOS 7 unsupported, it is explicitly _broken_ based on our use of C11 atomics and seccomp thread support and you will need to use docker to run there.

Reply:
Done